src/app/components/Slider/Slider.tsx

```diff
@@ -47,25 +51,23 @@ export const Slider: React.FC<SliderProps> = ({
               }}
               loop
               centeredSlides={true}
+              centeredSlidesBounds={true}
               //Parametry mogą być różne w zależności od slidera
-              spaceBetween={spaceBetween}
               slidesPerView={slidesPerView}
-              slidesOffsetBefore={170}
               initialSlide={initialSlide}
-              centeredSlidesBounds={true}
               allowTouchMove
             >
               {/* poglądowe, zamienić na dynamiczne  */}
 
               {partners
                 ? Array.from({ length: 8 }).map((_, index) => (
                     <SwiperSlide key={index} className="swiper__slide">
-                      <PartnersCard />
+                      <PartnersCard offset={offset}/>
                     </SwiperSlide>
                   ))
                 : Array.from({ length: 12 }).map((_, index) => (
                     <SwiperSlide key={index} className="swiper__slide">
-                      <StoriesCard />
+                      <StoriesCard offset={offset}/>
```

Comment:
Czemu to tutaj sa dwa komponenty "ParnersCard" i StoriesCard? czy one nie robia tego samego i nie powinny byc jednym komponentem?

Reply:
Dla tego, ze to 2 roznych komponenta :)
StoriesCard:
![image](https://github.com/SemperAvanti/semper-avanti/assets/147045521/80660259-62a9-4d26-84f4-178c38b11b50)
ParnersCard:
![image](https://github.com/SemperAvanti/semper-avanti/assets/147045521/ecce1d92-b10b-4efb-a133-d57c01788fd4)